### ISSUE TITLE
Feature request: 

### ISSUE THREAD

**Issue by madb1lly** [mozilla/addons#686]
Hi Add-ons team!

Please can you consider adding additional filters to narrow down results, e.g. if I'm looking at add-ons made by Mozilla (https://addons.mozilla.org/en-US/firefox/user/mozilla/) and I don't want to see language packs, only functionality extensions.

Thanks :)

**Comment by tofumatt** [comment]
Closing due to lack of information.

**Comment by madb1lly** [comment]
I accidentally hit return and it submitted the issue before I'd finished writing it, please reopen and reconsider.

**Comment by tofumatt** [comment]
Hey there!

No worries, we just sometimes get empty requests by mistake so I wasn't sure.

We'll be adding just user extensions and themes to user profiles in the new user profile page on the new site. I'll reference https://github.com/mozilla/addons-frontend/issues/2749 for now.

**Comment by madb1lly** [comment]
Yes, thanks for reopening, but since I didn't know if closed issues would be monitored I opened a new issue. In future I won't be so hasty!
Cheers :)
